### PR TITLE
completed css-layout-classes

### PR DESCRIPTION
**Pull request by marissalortega**
_Users_marissa ortega_repos_m1-0122-code-solutions_css-layout-classes_index html

### CODE REVIEW

File: css-layout-classes/index.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        <div class="column-one-fourth">
+          <img src="images/pikachu.png" alt="Pikachu">
+        </div>
+        <div class="column-one-fourth">
+          <img src="images/ness.png" alt="Ness">
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="half">
```
Comment (BrettAlbright) [inline css-layout-classes/index.html]: you want 2/3 for the text and 1/3 for the image

File: css-layout-classes/index.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            opponent in an industrial construction setting.
+          </p>
+        </div>
+        <div class="half">
+          <img src="images/dk.png" alt="Donkey Kong">
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="half">
```
Comment (BrettAlbright) [inline css-layout-classes/index.html]: same as above

File: css-layout-classes/styles.css [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-one-third {
+  width: 100%;
```
Comment (BrettAlbright) [inline css-layout-classes/styles.css]: remove all of the widths set here, you don't want to set both width and flex basis

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Looking good, a few small updates!

**Comment by marissalortega** [comment]
Screen Shot 2022-02-07 at 9 32 05 PM

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**